### DISCUSSION TITLE
add new addon source to instawow

### DISCUSSION THREAD

**Discussion by Char11e** [layday/instawow#152] [Q&A]
Hello everyone, I'm a newbie to instawow and also a translator for World of Warcraft addons.

https://addons.miliui.com/
The link above is the largest Chinese-language addon sharing forum.

I'm wondering what I need to do to get this website included as an addon source in instawow. Could you please advise me on this? Thank you.

**Comment by layday** [comment]
This appears to be an aggregator of some kind; I'd be very reluctant to add it as a default source.  However, instawow supports source plugins, so if you have any programming knowledge, you can write a plugin for this site.
